gems/pending/util/mount/miq_generic_mount_session.rb

```diff
@@ -457,14 +452,10 @@ def self.raw_disconnect(mnt_point)
     end
   end
 
-  def self.base_mount_point
-    case Sys::Platform::IMPL
-    when :macosx
-      "/Volumes"
-    when :linux
-      "/mnt"
-    else
-      raise "platform not supported"
-    end
+  private
+
+  def settings_mount_point
+    return if @settings[:mount_point].blank? # Check if settings contains the mount_point to use
```

Comment:
Yes.  If `Dir.mktmpdir` gets a nil for the second arg, it will use the default temp directory.